### PR TITLE
Wall lights now require light bulbs.

### PR DESCRIPTION
**Pull request by Zumorica**
Here's a [video](https://streamable.com/9zw2b) of it working.
You require any kind of gloves to take out the light tubes.

### CODE REVIEW

File: Resources/Textures/Objects/light_tube.rsi/meta.json [hunk 1]
```diff
@@ -0,0 +1,42 @@
+{
+    "version": 1,
+    "size": {
+        "x": 32,
+        "y": 32
+    },
```
Comment (RemieRichards) [inline Resources/Textures/Objects/light_tube.rsi/meta.json]: You should add ```"license"``` and ```"copyright"```, like so:
https://github.com/space-wizards/space-station-14-content/blob/74541e23a46a827d55de91943b9595f2c05d7a4e/Resources/Textures/Mob/human.rsi/meta.json#L7
https://github.com/space-wizards/space-station-14-content/blob/74541e23a46a827d55de91943b9595f2c05d7a4e/Resources/Textures/Mob/human.rsi/meta.json#L8

File: Content.Server/GameObjects/Components/Power/PoweredLightComponent.cs [hunk 2]
```diff
@@ -1,49 +1,171 @@
...
+
+        bool IAttackby.Attackby(IEntity user, IEntity attackwith)
+        {
+            if (!attackwith.HasComponent<LightBulbComponent>()) return false;
+
+            if (LightBulb != null) return false;
+
+            user.GetComponent<IHandsComponent>().Drop(attackwith, LightBulbContainer);
+
+            var inserted = LightBulbContainer.Insert(attackwith);
```
Comment (PJB3005) [inline Content.Server/GameObjects/Components/Power/PoweredLightComponent.cs]: Pretty sure this insert already gets done by the drop call, doesn't it?

Comment (Zumorica) [reply Content.Server/GameObjects/Components/Power/PoweredLightComponent.cs]: Yes, HandheldLightComponent does that too, so I thought it was necessary.

File: Resources/Textures/Objects/light_tube.rsi/meta.json (hunk 1 above)
Comment (Zumorica) [reply Resources/Textures/Objects/light_tube.rsi/meta.json]: Done!

**Pull request merged**